api/src/services/geo-service.ts

```diff
@@ -0,0 +1,412 @@
+    const { data } = await axios.get(url, { cancelToken: cancelTokenSource?.token });
+
+    return data;
+  }
+}
+
+/**
+ * Service for calling a Web Feature Service (WFS) API.
+ *
+ * @export
+ * @class WebFeatureService
+ * @extends {GeoService}
+ */
+export class WebFeatureService extends GeoService {
+  /**
+   * Executes a WFS GetCapabilities request.
+   *
+   * @param {SpatialService.OwsBaseOptions} [options]
+   * @return {*}  {Promise<unknown>}
+   * @memberof WebFeatureService
+   */
+  async getCapabilities(options?: SpatialService.OwsBaseOptions): Promise<unknown> {
+    const version = options?.version || '2.0.0';
+
+    const url = this._buildURL({
+      ...options,
+      request: 'GetCapabilities',
+      service: SpatialService.Wfs,
+      version: version
+    });
+
+    return this._externalGet(url);
+  }
+
+  /**
+   * Executes a WFS GetFeature request, returning features (exact response depends on the provided `options`).
+   *
+   * @param {(SpatialService.WfsGetBaseOptions | SpatialService.WfsGetFilterBaseOptions)} options
+   * @return {*}  {Promise<unknown>}
+   * @memberof WebFeatureService
+   */
+  async getFeature(
+    options: SpatialService.WfsGetBaseOptions | SpatialService.WfsGetFilterBaseOptions
+  ): Promise<unknown> {
+    const version = options?.version || '2.0.0';
+
+    const url = this._buildURL({ ...options, request: 'GetFeature', service: SpatialService.Wfs, version: version });
+
+    return this._externalGet(url);
+  }
+
+  /**
+   * Executes a WFS GetPropertyValue request, returning property values (exact response depends on the provided `options`).
+   *
+   * @param {(SpatialService.WfsGetBaseOptions | SpatialService.WfsGetFilterBaseOptions)} options
+   * @return {*}  {Promise<unknown>}
+   * @memberof WebFeatureService
+   */
+  async getPropertyValue(
+    options: SpatialService.WfsGetPropertyValueOptions | SpatialService.WfsGetPropertyValueFilterOptions
+  ): Promise<unknown> {
+    const version = options?.version || '2.0.0';
+
+    const url = this._buildURL({
+      ...options,
+      request: 'GetPropertyValue',
+      service: SpatialService.Wfs,
+      version: version
+    });
+
+    return this._externalGet(url);
+  }
+
+  /**
+   * Executes a WFS describeFeatureType request, returning a data element that describes the layer (names,
+   * types, properties, etc).
+   *
+   * Note: the BCGW
+   *
+   * @memberof WebFeatureService
+   */
+  async describeFeatureType() {}
+}
+
+/**
+ * Service for calling a Web Map Service (WMS) API.
+ *
+ * @export
+ * @class WebMapService
+ * @extends {GeoService}
+ */
+export class WebMapService extends GeoService {
+  /**
+   * Executes a WMS GetCapabilities request.
+   *
+   * @param {SpatialService.OwsBaseOptions} [options]
+   * @return {*}  {Promise<unknown>}
+   * @memberof WebMapService
+   */
+  async GetCapabilities(options?: SpatialService.OwsBaseOptions): Promise<unknown> {
```

Comment:
Nit: should be camel case to be consistent with the other class

Reply:
definitely, fixed.